src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+  .calendar {
+    flex-wrap: wrap;
+    gap: $calendar-gap;
+    width: calc(7 * $calendar-day-width + 7px);
```

Comment:
The `7px` value seems arbitrary and might not be consistent with the gap value defined by the `$calendar-gap` variable. Ensure that the total width calculation aligns with the design requirements and uses the appropriate variables for consistency.